### ISSUE TITLE
sqllite error in 1.8 series : `ValueError: error=OperationalError('near "dbm": syntax error')

### ISSUE THREAD

**Issue by David-Worboys** [Nuitka/Nuitka#2435]
Before submitting an Issue, please review the
[Issue Guidelines](https://github.com/Nuitka/Nuitka/blob/develop/CONTRIBUTING.md#submitting-an-issue).

- Please check whether the bug was already reported or fixed.

- Please check out if the develop version of Nuitka works better for you.

  Download source, packages [from here](https://nuitka.net/doc/download.html) where you will also
  find instructions how to do it via pip or git.

If you want to post a problem/bug, to help us understand and resolve your issue please check that
you have provided at least the information below, and discard up to here:

- Nuitka version, full Python version, flavor, OS, etc. as output by *this exact* command.

  > python -m nuitka --version
1.8.1
Commercial: None
Python: 3.11.4 (main, Jun  9 2023, 13:56:11) [GCC 11.3.0]
Flavor: Unknown
Executable: /home/david/PycharmProjects/311.4/bin/python
OS: Linux
Arch: x86_64
Distribution: Linuxmint (based on Ubuntu) 21.2
Version C compiler: /usr/bin/gcc (gcc).

  If you are not providing the full output. The issue cannot be solved and will be considered
  `invalid`. The command outputs more than you think, and we are adding more all the time. We do not
  want to left guessing or trying things out to reproduce your issue.

- How did you install Nuitka and Python
Loaded dependencies  from a requirements.txt file
Created a virtual environment from linux command line

  Did you use pip, anaconda, deb, rpm, from source, git clone, then install into a virtualenv or
  not, this is very important usually and one thing, the above command does not tell us (yet).

- The specific PyPI names and versions
pip 23.2.1

  It should be taken from this output if there specific packages involved, e.g. `numpy`, you are
  expected to shorten this to the relevant ones.

  > python -m pip freeze
```
appdirs==1.4.4
attrs==23.1.0
certifi==2023.7.22
cffi==1.15.1
charset-normalizer==3.2.0
cryptography==41.0.3
dill==0.3.7
fs==2.4.16
idna==3.4
lxml==4.9.3
multiprocess==0.70.15
Naked==0.1.32
netifaces==0.11.0
Nuitka==1.8.1
numpy==1.25.2
ordered-set==4.1.0
pathos==0.3.1
pathvalidate==3.1.0
platformdirs==3.10.0
pox==0.3.3
ppft==1.7.6.7
psutil==5.9.5
pycparser==2.21
pycryptodome==3.18.0
PyExecJS==1.5.1
PySide6==6.5.2
PySide6-Addons==6.5.2
PySide6-Essentials==6.5.2
python-dateutil==2.8.2
pytz==2023.3.post1
PyYAML==6.0.1
regex==2023.8.8
requests==2.31.0
shellescape==3.8.1
shiboken6==6.5.2
six==1.16.0
titlecase==2.4
tqdm==4.66.1
translators==5.8.3
typing==3.7.4.3
tzlocal==5.0.1
urllib3==2.0.4
xmltodict==0.13.0
zstandard==0.21.0

```

- Many times when you get an error from Nuitka, your setup may be special

  Then even a `print("hello world")` program will not work, please try that and report that error
  instead. We do not need a report that says "Tensorflow does not work" when nothing actually works
  for you. You ought to also read the User Manual and check that your setup is actually supported
  already.

- Also supply a Short, Self Contained, Correct, Example
```
All attempts at recreating in a minimal example failed.  

This is only happening on the 'full fat app' (https://github.com/David-Worboys/Black-DVD-Archiver)

Maybe something to do with antibloat?

This is a onefile compilation issue introduced in 1.8 relating to SQLite.

_Note this argument - --include-data-files=*.md=./ I added this in 1.8 and compile worked. It does not work in 1.81_
'''
python -m nuitka --lto=yes --python-flag=-OO --python-flag=no_warnings --python-flag=isolated  --verbose --verbose-output=./release/verbose.txt --disable-console --static-libpython=yes --standalone  --onefile --prefer-source-code --include-qt-plugins=sensible,multimedia  --assume-yes-for-downloads --show-anti-bloat-changes --enable-plugin=pyside6 --output-filename=black-dvd-archiver --output-dir=./release --linux-icon=/home/david/PycharmProjects/Black-DVD-Archiver/logo.jpg --include-data-files=*.svg=./ --include-data-files=*.png=./  --include-data-files=*.jpg=./ **--include-data-files=*.md=./** --include-data-dir=/home/david/PycharmProjects/Black-DVD-Archiver/tool_apps=./tool_apps --include-data-dir=/home/david/PycharmProjects/Black-DVD-Archiver/icons=./icons --include-data-dir=/home/david/PycharmProjects/Black-DVD-Archiver/IBM-Plex-Mono=./IBM-Plex-Mono  dvdarchiver.py

`ValueError: error=OperationalError('near "dbm": syntax error')
SELECT id, word FROM lang_tran WHERE word = 'no dbm clone found; tried ['dbm.gnu', 'dbm.ndbm', 'dbm.dumb']' AND language_code IS NULL
^CException ignored in: <module 'threading' (/tmp/onefile_177425_1694498535_258090/threading/threading.py)>
Traceback (most recent call last):
  File "/tmp/onefile_177425_1694498535_258090/threading.py", line 1583, in _shutdown
'''

```

  That demonstrates the issue i.e a small piece of code which reproduces the issue and can be run
  with out any other (or as few as possible) external dependencies. Issues without this may will
  likely be rejected without much consideration. Often this can be as simple as importing a package,
  if this is a packaging issue, try that first.

  Pointers to repositories with usage of `pipenv` are very welcome, archives with examples are too,
  e.g. if a certain package structure is needed. This must be only source code, binaries are not
  used ever by us. But beware, that e.g. we cannot click around and stuff. Still do an effort to
  make the usage obvious. Having a compile script in the repo is perfect.

  But this cannot be much code for non-commercial users, since that causes too much effort. We
  cannot just compile your code, run it and have it download a control and command center.

- Provide in your issue the Nuitka options used
`python -m nuitka --lto=yes --python-flag=-OO --python-flag=no_warnings --python-flag=isolated  --verbose --verbose-output=./release/verbose.txt --disable-console --static-libpython=yes --standalone  --onefile --prefer-source-code --include-qt-plugins=sensible,multimedia  --assume-yes-for-downloads --show-anti-bloat-changes --enable-plugin=pyside6 --output-filename=black-dvd-archiver --output-dir=./release --linux-icon=/home/david/PycharmProjects/Black-DVD-Archiver/logo.jpg --include-data-files=*.svg=./ --include-data-files=*.png=./  --include-data-files=*.jpg=./ **--include-data-files=*.md=./** --include-data-dir=/home/david/PycharmProjects/Black-DVD-Archiver/tool_apps=./tool_apps --include-data-dir=/home/david/PycharmProjects/Black-DVD-Archiver/icons=./icons --include-data-dir=/home/david/PycharmProjects/Black-DVD-Archiver/IBM-Plex-Mono=./IBM-Plex-Mono  dvdarchiver.py
`
  Ideally use the `# nuitka-project:` options feature in the code, so options and example code go
  along. Alternatively state the command line.

  [Nuitka Options in the code](https://nuitka.net/doc/user-manual.html#nuitka-options-in-the-code)

- Avoid unnecessary options

  Do not use `--onefile` if the issue also happens with `--standalone`. Minimize the use of options
  as far as you can, please. Do not disable outputs with `--quiet` and do not disable warnings.

- Note if this is a regression
`Yes, introduced 1.8`

  If it used to work in an earlier version of Nuitka, please note what you know about that. Since
  git bisect is a thing for which we do a lot of error to make it usable, this will help
  dramatically to isolate the issue.

- Consider getting commercial support

  [Nuitka commercial](https://nuitka.net/doc/commercial.html) offers subscriptions and priority
  support. This will accelerate your problem solution and helps to sustain Nuitka development. Also
  you then have a chance to provide source code of your project to us, which might simplify things,
  or pay for time to solve your issues in your environment.

Some things are not welcome, please consider it.

- Do *not* post screenshots

  These are not welcome unless absolutely necessary, e.g. because of Qt display problem, instead
  capture the output of programs, so things are searchable and copy&paste will work. I just plainly
  don't want to manually copy strings and hope they match.

- Do *not* close the issue yourself, we will close things on stable releases

  We close issues only when they are released as a stable version, e.g. in a hotfix or a new
  release, before that it will be "Done" in planning and go through `factory` and `develop` tags to
  indicate they are solved there.

  Of course, if you find out your issue is invalid, please do close it, and we then attach the
  `invalid` tag.

- Do *not* report against factory version

  Unless you were asked to test it there, it is frequently very broken, and there is only noise to
  be had. Telling me about it on Discord would be a better idea.

- Do *not* let this template remain part of the issue, it's noise.

**Comment by kayhayen** [comment]
You do not follow the issue template

> Do not let this template remain part of the issue, it's noise.

And you do not reduce options, nor do you attempt to minimize the issue, and then you truncated the call stack. I cannot work with this information unfortunaly.

**Comment by David-Worboys** [comment]
Fair enough Kay,

I posted that dogs breakfast after a very frustrating session trying to make a minimal reproduction and probably should have not posted..

Fell free to delete and I will post the issue again when I figured out how to minimally reproduce

**Comment by kayhayen** [comment]
Thanks
